app/uk/gov/hmrc/ngrnotifyproto/exporter/ExportEmailNotification.scala

```diff
@@ -67,27 +67,32 @@ class ExportEmailNotificationVOA @Inject() (
       // TODO Audit removal from queue
       emailNotificationRepo.delete(emailNotification._id).map(_ => ())
     else
-      logger.warn(s"Found ${emailNotification.trackerId} notification with send to ${emailNotification.sendToEmails}")
-      // TODO Audit send email
-      emailConnector
-        .sendEmailNotification(emailNotification)
-        .flatMap { res =>
-          res.status match {
-            case OK | ACCEPTED =>
-              emailNotificationRepo.delete(emailNotification._id).map(_ => ())
-            case BAD_REQUEST   =>
-              callbackConnector.callbackOnFailure(emailNotification, BAD_REQUEST_BODY, parseBadRequest(res.body))
-            case _             =>
-              callbackConnector.callbackOnFailure(
-                emailNotification,
-                WRONG_RESPONSE_STATUS,
-                s"Send email to user FAILED: ${res.status} ${res.body}"
-              )
+      for (sendTO <- emailNotification.sendToEmails)
```

Comment:
for (sendTO <- emailNotification.sendToEmails)
We don't need this loop. EmailConnector will send notification to all emails by single request with multiple emails in json body parameter "to"